### PR TITLE
Initial work on updating BSI to use d2l-navigation-bar-thick

### PR DESCRIPTION
**Pull request by jeroach**
Ready for initial review - Dave L suggested offline that some of this (e.g. logo-area should move back to BSI).
- removing style code that has moved to d2l-navigation

### CODE REVIEW

**Comment by jeroach** [comment]
Fixing the reference issues (basically adding some variables back - for now).

File: sass/navigation/common.scss [hunk 1]
```diff
@@ -1,40 +1,35 @@
 @import '../../bower_components/d2l-colors/d2l-colors.scss';
-@import '../common.scss';
 
-$d2l-navigation-header-height: 90px;
 $d2l-navigation-header-height-mobile: 72px;
```
Comment (svanherk) [inline sass/navigation/common.scss]: Is this still used somewhere?

Comment (jeroach) [reply sass/navigation/common.scss]: Yes, https://github.com/Brightspace/brightspace-integration/blob/master/sass/navigation/mobile-menu.scss#L68

File: sass/navigation/main.scss [hunk 2]
```diff
@@ -134,10 +134,6 @@
 	}
 }
 
-.d2l-navigation-s-gutters {
```
Comment (svanherk) [inline sass/navigation/main.scss]: I think the Github comparison being shown here is stale, in master I'm seeing lines in the file that aren't showing up here (like https://github.com/Brightspace/brightspace-integration/blob/master/sass/navigation/main.scss#L4).

Comment (jeroach) [reply sass/navigation/main.scss]: I'll merge master into my branch to refresh

**Comment by jeroach** [comment]
FYI CI will continue to fail until the `navigation` PR is merged.

File: sass/navigation/common.scss [hunk 3]
```diff
@@ -1,40 +1,35 @@
...
 
-.d2l-navigation-s-centerer {
+.d2l-navigation-s-centerer { // remove once bottom nav bar is in d2l-navigation
 	margin: 0 auto;
 	max-width: 1230px;
 }
 
-.d2l-navigation-s-gutters {
+.d2l-navigation-s-gutters { // remove once bottom nav bar is in d2l-navigation
 	@include d2l-page-padding();
```
Comment (svanherk) [inline sass/navigation/common.scss]: I'm guessing this is what we will replace with that "page sizer component", that controls whether the navbar and/or page is full width or that margin-ed width.  I don't think we have a story for that, unless we consider it part of the main navbar story.

**Comment by jeroach** [comment]
@svanherk This is ready for final review

File: bower.json [hunk 4]
```diff
@@ -2,7 +2,6 @@
   "name": "brightspace-integration",
   "dependencies": {
     "1.x-hybrid-components": "https://github.com/BrightspaceHypermediaComponents/1.x-hybrid-components.git#a6173a828de1c2566236a88d6932f2485400b7ac",
-    "Sortable": "https://github.com/Brightspace/Sortable.git#a8d96e457b2c3bc49adc7cc2db2088f75dd24fb0",
```
Comment (svanherk) [inline bower.json]: Minor: The sort order of this one changes depending on Windows versus Mac machines, so we tend to just leave these changes out of the commit

File: sass/navigation/header.scss [hunk 5]
```diff
@@ -35,7 +19,7 @@
 	overflow: hidden;
 	padding: 0 7px;
 
-	.d2l-navigation-s-header-container & {
+ 	.d2l-navigation-header-container & {
```
Comment (svanherk) [inline sass/navigation/header.scss]: Here we are removing the "s", but it was left here: https://github.com/Brightspace/brightspace-integration/pull/921/files#diff-147e14480bb8225a10daf6352f7759dcR86.  Should we be changing it there as well?  Looks like we change the only two previous uses of it: http://search.dev.d2l/source/search?q=%22d2l-navigation-s-header-container%22&defs=&refs=&path=&hist=&type=&project=Lms.

File: bower.json (hunk 4 above)
Comment (jeroach) [reply bower.json]: I'll work on removing these changes

File: sass/navigation/header.scss (hunk 5 above)
Comment (jeroach) [reply sass/navigation/header.scss]: It should be removed in the other spot. Artifact of merging. I'll fix. Thanks.

File: sass/navigation/main.scss [hunk 6]
```diff
@@ -143,7 +143,7 @@
...
+.d2l-navigation-s-gutters { // remove once bottom nav bar is in d2l-navigation
```
[the remark this answers is not in the document]
Comment (jeroach) [reply sass/navigation/main.scss]: No, this is still needed as is for the bottom nav-bar.

The style reference for `d2l-navigation-gutters` is in the `d2l-navigation-shared-styles.html` file of the `navigation` repo.

[the remark this answers is not in the document]
Comment (svanherk) [reply sass/navigation/main.scss]: Sorry, I commented on the file but it has nothing to do with this actual line.  It's line 4 of this file, around the no-links styles.  I'm seeing the "s" removed in your PR in `NavigationView`, but not in `NoLogin` (https://git.dev.d2l/projects/CORE/repos/platformtools/browse/navbars/D2L.PlatformTools.Navbars/Web/Desktop/Rendering/Views/Daylight/NoLogin.Renderer.cs?at=01818ba5fcddb7262eb409228d0ae67256e651c0#18)

[the remark this answers is not in the document]
Comment (jeroach) [reply sass/navigation/main.scss]: Stash is down due to DNS.... 😢

[the remark this answers is not in the document]
Comment (jeroach) [reply sass/navigation/main.scss]: what is the class name in question? looking at my local copy of the `NoLogin.Renderer.cs` I don't see any concerns.

[the remark this answers is not in the document]
Comment (svanherk) [reply sass/navigation/main.scss]: `d2l-navigation-s-main-no-links`

[the remark this answers is not in the document]
Comment (jeroach) [reply sass/navigation/main.scss]: I think that the change needs to be to `NavigationView.Render.cs` (e.g. put back the `s` to make that class name also `d2l-navigation-s-main-no-links`). Good catch.

However, I don't really trust myself right now, and I need to take some Advil... I also need to move my desk again.

I'll double check and try to get this sorted tomorrow morning. Thanks for reviewing and sorry for all the back and forth...this isn't one of my better days...

[the remark this answers is not in the document]
Comment (svanherk) [reply sass/navigation/main.scss]: No problem, I was trying to review while in another meeting so my comments definitely could've been clearer.  I forgot it was moving day... sounds not fun.

[the remark this answers is not in the document]
Comment (jeroach) [reply sass/navigation/main.scss]: Okay, other PR fixed and CI triggered: https://git.dev.d2l/projects/CORE/repos/platformtools/pull-requests/2526/overview

[the remark this answers is not in the document]
Comment (jeroach) [reply sass/navigation/main.scss]: Took a while to setup my desk this morning but I'm finally up and running 😄

File: sass/navigation/common.scss [hunk 7]
```diff
@@ -1,39 +1,36 @@
 @import '../../bower_components/d2l-colors/d2l-colors.scss';
-@import '../common.scss';
```
Comment (svanherk) [inline sass/navigation/common.scss]: Why are we able to remove this?  Is it used to get the `d2l-page-padding` mixin?

Comment (jeroach) [reply sass/navigation/common.scss]: Correct.

File: sass/navigation/header.scss [hunk 8]
```diff
@@ -1,22 +1,6 @@
 @import '../../bower_components/d2l-colors/d2l-colors.scss';
 @import 'common.scss';
 
-.d2l-navigation-s-gutter {
```
Comment (svanherk) [inline sass/navigation/header.scss]: Looks like this is still used here? http://search.dev.d2l/source/xref/Lms/platformTools/navbars/D2L.PlatformTools.Navbars/Web/Desktop/Rendering/Views/Daylight/MobileMenu.Renderer.cs#214

File: sass/navigation/common.scss (hunk 7 above)
Comment (jeroach) [reply sass/navigation/common.scss]: Wait - the `d2l-page-padding` mixin is needed. It wasn't originally. I'll add the import back.

File: sass/navigation/header.scss (hunk 8 above)
Comment (jeroach) [reply sass/navigation/header.scss]: Yeah it appears that way - thought I checked all the references when I first pulled it out. I'll add it back (and remove the copied versions in the navigation code).

There will be another PR for `navigation` to accommodate this.

**Comment by jeroach** [comment]
@svanherk Fixed your latest comments and also tidied up the PR a bit - should be much cleaner now.

**Pull request merged**